src/test/java/se/sundsvall/messaging/api/StatusAndHistoryResourceFailureTest.java

```diff
@@ -99,7 +99,7 @@ void getMessageStatusShouldFailWithInvalidUuId() {
 
 		// Act
 		final var response = webTestClient.get()
-			.uri(uriBuilder -> uriBuilder.path(MESSAGE_STATUS_PATH).build(Map.of("messageId", messageId)))
+			.uri(uriBuilder -> uriBuilder.path("/2281" + MESSAGE_STATUS_PATH).build(Map.of("messageId", messageId)))
```

Comment:
Shouldn't MESSAGE_STATUS_PATH (and all other paths in this class) be updated with municipalityId?